### DISCUSSION TITLE
16.4.11

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#891] [General]
## Releases Notes for 16.4.11
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/draw.io-16.4.11-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/draw.io-16.4.11-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/draw.io-universal-16.4.11.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/drawio-amd64-16.4.11.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/drawio-x86_64-16.4.11.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v16.4.11/drawio-x86_64-16.4.11.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Fixes #873 , #860 
- Updates to [draw.io core 16.4.11](https://github.com/jgraph/drawio/blob/v16.4.11/ChangeLog). All changes from 16.4.1 to 16.4.11, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v16.4.11'>16.4.11</a>.</em>

**Comment by siztem3rror** [comment]
Was removing the templates from 'new' intentional?

![image](https://user-images.githubusercontent.com/36902728/151572294-da3b34a5-2493-4f98-8a77-615ccbf0ab43.png)

**Reply by davidjgraph** [reply]
Fixed in 16.5.1. Thanks for report.
